api/enterprise-api-ref/build-api.markdown

```diff
@@ -76,11 +76,42 @@ HTTP 200 Ok
 
 **Responses:**
 
-| HTTP response code | Description |
-|--------|-------------|
-| 200 OK | Project successfully created |
-| 422 Unprocessable entity | Validation error occurred |
-| 500 Internal server error | Internal server error |
+| HTTP response code        | Description                  |
+|---------------------------|------------------------------|
+| 200 OK                    | Project successfully created |
+| 422 Unprocessable entity  | Validation error occurred    |
+| 500 Internal server error | Internal server error        |
+
+## Create empty project
+
+You can create an empty project without repository data and the project will exist only locally.
+This type of project is useful for testing purposes.
+
+**URI:** https://hub.cfengine.com/api/build/projects/empty
```

Comment:
I would guess that after this ticket, it was not called empty any more? https://tracker.mender.io/browse/ENT-9460

Reply:
ok, I will change it, to https://hub.cfengine.com/api/build/projects/local, they both are working

Reply:
done